### PR TITLE
OSDOCS-7766: Migrate "Enabling the AWS EFS CSI Driver Operator on ROSA" to ROSA product documentation

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-7766](https://issues.redhat.com//browse/OSDOCS-7766): Migrate "Enabling the AWS EFS CSI Driver Operator on ROSA" to ROSA product documentation

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-7766

Link to docs preview:
https://64868--docspreview.netlify.app/openshift-rosa/latest/rosa_tutorials/rosa-mobb-enabling-aws-efs-csi-driver-operator-rosa

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: rosa_tutorials/rosa-mobb-enabling-aws-efs-csi-driver-operator-rosa.adoc
Comment (tmalove) [inline rosa_tutorials/rosa-mobb-enabling-aws-efs-csi-driver-operator-rosa.adoc:446]:
```suggestion
It might take a few minutes for the pod to be ready. If you see errors such as `Output: Failed to resolve "fs-XXXX.efs.us-east-2.amazonaws.com"`, it likely means it is still setting up the EFS volume.
```

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
@jneczypor  Don't forget to squash your commits before the merge request...looks good with a few suggestions (some for the next round). Thanks!

**Comment by jneczypor** [comment]
Hi @michaelryanmcneill - Can I please have a review of this PR? Thanks so much!

**Comment by duanwei33** [comment]
Hello, I 'm OCP storge QE and we are just reviewing ROSA & AWS EFS CSI Driver Operator in https://github.com/openshift/openshift-docs/pull/64612, may I know what is the difference between them?

Also I'm not sure do we need this section under Tutorials while we have the EFS operator installation in following section?
https://64868--docspreview.netlify.app/openshift-rosa/latest/storage/container_storage_interface/osd-persistent-storage-aws-efs-csi

cc @ropatil010

**Comment by michaelryanmcneill** [comment]
/hold

In agreement with @duanwei33: I'm not confident it makes sense to put this in Tutorials. I'm going to verify the existing doc matches the tutorial doc. If so, we'll close this in favor of that.

**Pull request closed without merging**
